airflow/decorators/__init__.pyi

```diff
@@ -410,5 +412,12 @@ class TaskDecoratorCollection:
             of the target ConfigMap's Data field will represent the key-value
             pairs as environment variables. Extends env_from.
         """
+    @overload
+    def sensor(self, **kwargs) -> TaskDecorator:
+        """
+        Wraps a Python function into a sensor operator.
+        """
```

Comment:
Can you list out more arguments and the docstring explicitly (like other functions above)? This is shown in editor autocomplete, and the current format is not useful.